docker/nginx-custom.conf

```diff
@@ -0,0 +1,38 @@
+server {
+    listen       8080;
+    # same config for localhost, dev and prod environments
+    server_name  localhost 
+                swedgene.scilifelab-2-dev.sys.kth.se
+                genomes.scilifelab.se;
+    absolute_redirect off;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+    }
+
+    error_page  404              /404.html;
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
+        root   /usr/share/nginx/html;
+    }
+
+    # Cache media files 
```

Comment:
You could combine this since it has the same rules.
like this:
location ~* \.(webp|svg|jpg|jpeg|png|gif|woff|woff2|ttf|eot|otf)$ {

or is there a specific reason?

Reply:
Fair point, I have combined them now as part of commit 7500e00953d8cfd91204e9ff65c5230533ecb59f